qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
Can we please avoid the use of magic numbers here? The easiest way to do this is using the function `get_count` to know how many are in the table already and then you just need to add 1 to know the new id. This will make future changes really easy :-)